src/main/docker/Dockerfile

```diff
@@ -1,63 +1,70 @@
-FROM debian:stable-slim
-LABEL maintainer="steve.springett@owasp.org"
-LABEL vendor="OWASP"
+FROM eclipse-temurin:11.0.13_8-jre-focal AS jre-build
```

Comment:
We should go ahead and jump to `17.0.2_8-jre-focal` directly.

Reply:
Oh okay, so directly to the new LTS? 
I have heard of ppl running into issues at work who compiled against 11 and ran on 17.
Just want to throw out this as a note, but I will change it if you still want me to.

Reply:
@k3rnelpan1c-dev I've been testing with Java 17, but the testing has not been extensive. Lets hold off on it for now. I'm concerned about some of the changes to reflection.